slave/centos-7/cloud-init-base.sh

```diff
@@ -39,6 +45,12 @@ yum install -y \
 	curl \
 	enchant
 
+# Install whatever version is newest in the Docker repository.  If you wanted a
+# different version, you should have run the script at a different time.
```

Comment:
What is important, I think, is that the way we install Docker here matches some docs somewhere about how users should install Docker. It is a shame that we are duplicating code, but for now add a comment (in both places, here and where the Docker install instructions are in the docs?) that this should match some particular docs.

(Same goes for Ubuntu)